docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -4,89 +4,92 @@
 
 beta::[]
 
-{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
-of jobs to detect anomalies with certain conditions. If an anomaly meets the 
-conditions, the alert triggers the defined action. For example, you can create 
-an alert that checks an {anomaly-job} every fifteen minutes for critical 
-anomalies and notifies you in an email. This page helps you to configure an 
-{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{anomaly-detect-cap} alerting runs scheduled checks on an {anomaly-job} or a 
```

Comment:
Since [https://www.elastic.co/guide/en/kibana/master/alerting-getting-started.html#alerting-concepts-differences](Kibana alerting docs) already describe "Kibana alerting" vs "Elasticsearch alerting", I think we should try to avoid the idea that there's another type of "ML alerting" or "anomaly detection alerting".  If you agree, I suggest something like this: 
```suggestion
{kib} {alert-features} include support for {ml} rules, which runs scheduled checks on an {anomaly-job} or a 
```